File: Player.py
class Player:
    def __init__(self, player):
        self.firstName = player["firstName"]
        self.lastName = player["lastName"]
        self.pos = player["pos"]
        self.jersey = player["jersey"]

    def get_first_name(self):
        return self.firstName

    def get_last_name(self):
        return self.lastName



# TODO: I want to conver player to Jason but it makes {property: "value"}
# so i must to use the json key as property and its bad convention like firstName

File: src/Player.ts
class Player{
    private firstName:string
    private lastName:string
    private jersey:string
    private pos:string
    private img:string
    private isInDreanTeam:boolean

    constructor(firstName:string, lastName:string, jersey:string, pos:string){
        this.firstName = firstName
        this.lastName = lastName
        this.jersey = jersey
        this.pos = pos
        this.img = `https://nba-players.herokuapp.com/players/${lastName}/${firstName}`
        this.isInDreanTeam = false
    }

    setIsInDreamTeam(flag: boolean){
        this.isInDreanTeam = flag
    }

    getFirstName() : string{
        return this.firstName
    }

    getLastName() : string{
        return this.lastName
    }

    
    // get irstName(): string {
    //     return this.firstName;
    // }

    // get astName(): string {
    //     return this.lastName;
    // }




}